docs/basics/object-spawning.md

```diff
@@ -21,11 +21,11 @@ When a `NetworkBehaviour` is assigned to a `NetworkObject`, the `NetworkObject.N
 
 ### Registering a Network Prefab
 
-One of the requirements to be able to spawn a network Prefab instance is that it must be registered with the `NetworkManager` via the "Network Prefabs" property list.
-The two steps to registering a network Prefab with `NetworkManager`:
+You must register a Network Prefab instance with a `ScriptableObject` via the Network Prefabs property list.
```

Comment:
See my message above.

`ScriptableObject` is a very generic thing. It's a specific type of `ScriptableObject` called `NetworkPrefabsList`. And we register all prefabs to it by default now, but it does still have to be registered with one.

Reply:
I'll refrain from commenting on further instances where this needs to be clarified, but I see that there are others.